### DISCUSSION TITLE
Adding a itemized list in a multi row/column table cell

### DISCUSSION THREAD

**Discussion by russurquhart1** [lvjr/tabularray#519] [Q&A]
Hi,

Based on the example in the Tabularray docs, I created a multi row column example, and tried to put an itemized list in one of the cells:

```
\documentclass[landscape]{article}
\usepackage{tabularray}
\usepackage{xcolor}

\begin{document}

%test of a table

\begin{tblr}{|l|l|l|l|l|}
\hline
\SetCell[r=2]{l} 2 Rows
   & \SetCell[c=2]{l} 2 Columns
   &   & \SetCell[r=2,c=2]{l} This is a list
   \begin{itemize}
   \item This is a bullet.
   \item Another bullet. 
   \item Another bullet.
   \end{itemize} &  \\
\hline
 & 2-2 & 2-3 & & \\
\hline
\end{tblr}

\end{document}
```

In the version of MiKTeX that I'm using it gives this error message:

```
! LaTeX Error: Something's wrong--perhaps a missing \item.

See the LaTeX manual or LaTeX Companion for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.22 \end
         {tblr}
```

I loaded the same file into Overleaf. I compiled correctly, but gave similar error messages. It implied that you could NOT have a itemized list in a table cell. Further in the Tabularray docs, they show a sample of a bullted list in a table cell. Does LaTeX not allow lists in a table cell?

Thanks for any help!

Russ

**Comment by muzimuzhi** [comment]
> Further in the Tabularray docs, they show a sample of a bullted list in a table cell.

To use list-like stuff in table cells, `measure=vbox` (inner) option, provided by the `varwidth` library is a must. See package manual (v2024A, 2024-02-16), sec. 5.10 Library `varwidth` for more info.

```tex
\documentclass[landscape]{article}
\usepackage{tabularray}
\usepackage{xcolor}

\UseTblrLibrary{varwidth}

\begin{document}

%test of a table

\begin{tblr}{colspec={lllll}, hlines, vlines, measure=vbox}
  \SetCell[r=2]{l} 2 Rows
     & \SetCell[c=2]{l} 2 Columns
     &   & \SetCell[r=2,c=2]{l} This is a list
     \begin{itemize}
     \item This is a bullet.
     \item Another bullet. 
     \item Another bullet.
     \end{itemize} &  \\
     & 2-2 & 2-3 & & \\
\end{tblr}

\end{document}
```

![image](https://github.com/user-attachments/assets/c996dc65-9602-45a9-9b0a-be09725d50d3)

**Reply by russurquhart1** [reply]
Thank you very much, this was the missing piece I needed for my documents!!

I also found and saw the reference you mentioned. As a technical writer, could I possibly suggest that I think it would be helpful if you put a Note in at the end of Section 1.5 that linked/referenced Section 5.8 as a method to add lists to cells? Just a thought!

Thanks again for all your help!
